api/internal/localizer/locloader.go

```diff
@@ -116,24 +113,30 @@ func (ll *locLoader) Load(path string) ([]byte, error) {
 // New returns a Loader at path if path is a valid localize root.
 // Otherwise, New returns an error.
 func (ll *locLoader) New(path string) (ifc.Loader, error) {
-	repoSpec, err := git.NewRepoSpecFromURL(path)
-	if err == nil && repoSpec.Ref == "" {
-		return nil, errors.Errorf("localize remote root '%s' missing ref query string parameter", path)
-	}
-
 	ldr, err := ll.Loader.New(path)
+	// timeout indicates path is a root, not a file
+	if utils.IsErrTimeout(err) {
```

Comment:
Note to self: Add TODO statement to refactor once appropriate [errors added to `ifc.Loader`](https://github.com/kubernetes-sigs/kustomize/issues/4807#issuecomment-1256411646)